python/cells.cpp

```diff
@@ -690,6 +690,9 @@ void register_cells(pybind11::module& m) {
             "while the method for calculating reversal potential is global for all\n"
             "compartments in the cell, and can't be overriden locally.")
         // Paint mechanisms.
+        .def("paintings",
```

Comment:
Btw, is this supposed to be part of this PR?

Reply:
Errr. No.